src/main/java/net/dv8tion/jda/internal/entities/EntityBuilder.java

```diff
@@ -931,7 +931,7 @@ public static Activity createActivity(DataObject gameJson)
 
         if (type == Activity.ActivityType.CUSTOM_STATUS)
         {
-            if (gameJson.hasKey("state") && name.equalsIgnoreCase("Custom Status"))
+            if (gameJson.hasKey("state"))
```

Comment:
Why remove this?

Reply:
This is an implementation detail that the client itself does not rely on and other libraries do not respect (djs allows you to set anything for name, for example.) I figured it was better for reliability to remove it.

Reply:
What does the client do if you set the name to `Custom Status` vs anything else?

Reply:
It renders it the same -- it's just ignored.